### PR TITLE
Extended settings for Aqara dual relay module T2 and Aqara LED strip T1

### PR DESCRIPTION
**Pull request by kirovilya**

### CODE REVIEW

File: src/devices/xiaomi.ts [hunk 1]
```diff
@@ -2524,14 +2602,41 @@ const definitions: Definition[] = [
...
         description: 'Aqara dual relay module T2',
-        fromZigbee: [fz.on_off, fz.aqara_opple, fz.xiaomi_power],
-        toZigbee: [tz.on_off],
-        meta: {multiEndpoint: true},
+        fromZigbee: [fz.on_off, fz.aqara_opple, fz.xiaomi_power, fzLocal.attributes, fzLocal.attributes_multistate],
+        toZigbee: [tz.on_off, tzLocal.attributes],
+        meta: {
+            multiEndpoint: true,
+            attributes: {
+                'switch_type': [1, 'aqaraOpple', 0x000a, 0x20, {'toggle': 1, 'momentary': 2, 'none': 3}],
```
Comment (Koenkk) [inline src/devices/xiaomi.ts:2610]: There are already converters available for this which are used by multiple xiaomi devices, no need to add new ones:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/085c42e5da15e28a60293182fed56e72577ec041/src/converters/toZigbee.js#L2267

File: src/devices/xiaomi.ts [hunk 2]
```diff
@@ -2524,14 +2602,41 @@ const definitions: Definition[] = [
...
+        },
         endpoint: (device) => {
             return {'l1': 1, 'l2': 2};
         },
         exposes: [e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'),
             e.power(), e.current(), e.energy(), e.voltage(), e.device_temperature(),
+            e.enum('switch_type', ea.ALL, ['toggle', 'momentary', 'none']).withLabel('Switch type')
+                .withDescription('External switch type'),
+            e.binary('interlock', ea.ALL, true, false)
+                .withDescription('Enabling prevents both relais being on at the same time (Interlock)'),
```
Comment (craiggenner) [inline src/devices/xiaomi.ts:2627]: small spelling correction - `relais` => `relays`

**Comment by Koenkk** [comment]
I will move this to `toZigbee.ts`, will try to wrap up https://github.com/Koenkk/zigbee-herdsman-converters/pull/6299 ASAP

**Comment by kirovilya** [comment]
closed by https://github.com/Koenkk/zigbee-herdsman-converters/pull/6554

**Pull request closed without merging**